static/app/views/organizationGroupDetails/groupSimilarIssues/similarStackTrace/list.tsx

```diff
@@ -50,58 +38,61 @@ class List extends Component<Props, State> {
       </PanelBody>
     </Panel>
   );
+}
 
-  handleShowAll = () => {
-    this.setState({showAllItems: true});
-  };
-  render() {
-    const {orgId, groupId, project, items, filteredItems, pageLinks, onMerge, v2} =
-      this.props;
-
-    const {showAllItems} = this.state;
-
-    const hasHiddenItems = !!filteredItems.length;
-    const hasResults = items.length > 0 || hasHiddenItems;
-    const itemsWithFiltered = items.concat((showAllItems && filteredItems) || []);
-
-    if (!hasResults) {
-      return this.renderEmpty();
-    }
-
-    return (
-      <Fragment>
-        <Header>
-          <SimilarSpectrum />
-        </Header>
-
-        <Panel>
-          <Toolbar v2={v2} onMerge={onMerge} />
-
-          <PanelBody>
-            {itemsWithFiltered.map(item => (
-              <Item
-                key={item.issue.id}
-                orgId={orgId}
-                v2={v2}
-                groupId={groupId}
-                project={project}
-                {...item}
-              />
-            ))}
-
-            {hasHiddenItems && !showAllItems && (
-              <Footer>
-                <Button onClick={this.handleShowAll}>
-                  {t('Show %s issues below threshold', filteredItems.length)}
-                </Button>
-              </Footer>
-            )}
-          </PanelBody>
-        </Panel>
-        <Pagination pageLinks={pageLinks} />
-      </Fragment>
-    );
+function List({
+  orgId,
+  groupId,
+  project,
+  items,
+  filteredItems = [],
+  pageLinks,
+  onMerge,
+  v2,
+}: Props) {
+  const [showAllItems, setShowAllItems] = useState<boolean>(false);
```

Comment:
since the type automatically interferes as it contains the default value false, do we still want to define the type?